docs/develop/guides/cheat-sheets/momento-cache-python-cheat-sheet.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 1
+sidebar_label: Python
+sidebar_class_name: sidebar-item-python-cheat-sheet
+title: Cheat Sheet for Python + Momento Serverless Cache
+description: Everything to get you going with coding using Python and Momento Serverless Cache
+---
+
+# Cheat Sheet for Python with Momento Serverless Cache
+If you need to get going quickly with Python and Momento Serverless Cache, here are the basic API calls you'll need. When you need them, check out the [API reference page](../../api-reference/index.mdx) for more information on the full assortment of Momento API calls.
+
+If you combine all of the functions on this page into one python file, you'd have a central collection of functions you can import and call from other python code. In addition, if you are putting this into production, you might look to replace the print() calls with ones using the logging library (`import logging`) in python.
+
+## Import libraries and connect to return a CacheClient object
+```python
+from datetime import timedelta
+from momento import CacheClient, Configurations, CredentialProvider
+from momento.responses import CacheGet, CacheSet, CreateCache, ListCaches, CacheIncrement
+
+import os
+
+  def client():
+    momento_auth_token = CredentialProvider.from_environment_variable('MOMENTO_AUTH_TOKEN')
+    ttl  = timedelta(seconds=int(os.getenv('MOMENTO_TTL_SECONDS', '60')))
+    config = {
+      'configuration': Configurations.Laptop.v1(),
+      'credential_provider': momento_auth_token,
+      'default_ttl':  ttl
+    }
+    return CacheClient(**config)
+```
+
+## Create a new cache in Momento Serverless Cache
+Use this function to create a new cache in your account
+```python
+  def create_cache(cache_name):
+    with MomentoCounter.client() as client:
```

Comment:
I don't see where `MomentoCounter` is defined here. Did you mean just `client()`?

Reply:
MomentoCounter is the name of the class these functions are a part of...and yeah, I omitted that. i think I need to put a link to the entire class file to refer to.